starky/src/lib.rs

```diff
@@ -1,7 +1,6 @@
 #![allow(incomplete_features)]
```

Comment:
```suggestion
```

I believe this can also be removed now :)

Reply:
You're right! Thanks for the catch